genesis/engine/solvers/rigid/rigid_solver_decomp.py

```diff
@@ -1174,13 +1174,19 @@ def _init_equality_fields(self):
         )
         if self.n_equalities > 0:
             equalities = self.equalities
+
+            equalities_sol_params = np.array([equality.sol_params for equality in equalities], dtype=gs.np_float)
+            equalities_sol_params = _sanitize_sol_params(
+                equalities_sol_params, self._substep_dt, self._sol_contact_resolve_time
```

Comment:
You should not specify `self._sol_contact_resolve_time` because equality constraints are NOT contact. Just leave this field empty: `_sanitize_sol_params(equalities_sol_params, self._substep_dt)`

Reply:
add a warning "contact_resolve_time is deprecated. Please use constraint_resolve_time instead."